src/ImageFitter/ImageFitter.cc

```diff
@@ -141,13 +141,19 @@ void ImageFitter::StopFitting() {
     _fit_data.stop_fitting = true;
 }
 
-void ImageFitter::CalculateNanNum() {
+void ImageFitter::CalculateNanNumAndStd() {
+    std::vector<double> data_notnan;
```

Comment:
Thanks for the comment. `reserve` is added.

The size of the data is an image channel. Yes, it can be large with large images, but it should be temporary and the workspace `gsl_multifit_nlinear_workspace` used in the fitting process is larger than it.